### PR TITLE
added custom protocol documentation

### PR DESCRIPTION
**Pull request by adamdubiel**

### CODE REVIEW

**Comment by wojtkiewicz** [comment]
👍

File: docs/docs/configuration/custom-protocol.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+});
+```
+
+The `HermesConsumerBuilder#withMessageSenderProvider` method accepts a lazy evaluated function which gives access to
+containers `ServiceLocator`. In some more advanced cases it might be beneficial to use some internal components,
+however there are no compatibility guarantees.
+
+### Extending HTTP message sender
+
+In case you would like to build on existing implementation of HTTM messages sender, it can be created and registered
```
Comment (dankraw) [inline docs/docs/configuration/custom-protocol.md]: HTTM -> HTTP

**Comment by dankraw** [comment]
👍 (+ comment)

**Comment by adamdubiel** [comment]
fixed, merging after build goes green

**Pull request merged**